R/RtGam.R

```diff
@@ -109,6 +133,7 @@ RtGam <- function(cases,
     fit = fit,
     df = df,
     group = group,
+    day_of_week = day_of_week,
```

Comment:
'day of week' is in the df, so does it make sense to pass it again here?

Reply:
Yes -- this is returning the user-inputted value not the parsed value in the dataframe.